.cirrus.yml

```diff
@@ -178,8 +178,7 @@ build_task:
               DISTRO_NV: ${UBUNTU_NAME}
               VM_IMAGE_NAME: ${UBUNTU_CACHE_IMAGE_NAME}
               CTR_FQIN: ${UBUNTU_CONTAINER_FQIN}
-              # FIXME 2022-07-12: change to runc once #14833 is fixed!
-              CI_DESIRED_RUNTIME: crun
+              CI_DESIRED_RUNTIME: runc
```

Comment:
@edsantiago I don't remember checking this or not, so maybe a duplicate comment:  In `lib.sh` there is a list of "pass-through" env. var. names.  Please make sure this one is on that list to ensure the variable and value are passed into (for example) rootless and podman-in-podman CI scripts.

Reply:
That's a really good point, thanks. It looks like the test [passed in int-rootless](https://api.cirrus-ci.com/v1/artifact/task/5577721127370752/html/int-podman-fedora-36-rootless-host.log.html#t--Podman-info--check-runtime--1), but not in containerized, so I have work to do. I've also added a triple-check so the test will fail if `$CIRRUS_CI` is defined but `$CI_DESIRED_RUNTIME` is not.

Reply:
Oof, one thing to watch out for here is: I don't think we set `CIRRUS_CI=true` or `CI=true` in `hack/get_ci_env.sh`.  However that script automatically runs `setup_environment.sh` for developers before handing them a shell.  So checks against those vars could cause problems.

BTW, have you tried `hack/get_ci_vm.sh` recently?  For a number of months,  it has been updated to not require `sudo`, only runs rootless, and only touches files under your `~/.config/gcloud` directory.  All things you took issue with in the past.  Anyway...in case it's useful for you.  It's certainly faster for debugging CI changes than waiting for CIrrus to run them.

Reply:
I did try `get_ci_vm.sh` 2-3 weeks ago, rootless (thank you!), but it failed with a long complicated error that I couldn't figure out. I tried a reset(?) arg, still failed. Gave up.

It's 100% OK if `$CIRRUS_DESIRED_RUNTIME` is set without `$CIRRUS_CI`. What is not OK is the converse. And I think (fingers crossed) that this iteration, the one running in CI just now, handles all the corner cases. We'll see in two hours.

Reply:
>  failed with a long complicated error

That may have been the SNAFU w/ AWS.  Should be fixed now.

> I tried a reset(?) arg, still failed. Gave up.

You can try completely removing `~/.config/gcloud` worst-case.  Everythnig in there is generated automatically.  All you'll have to do is re-init (do the browser URL copy-paste thing, select region, etc.).